### ISSUE TITLE
'openssl speed' | Non-operational support of multiple algorithms in command-line

### ISSUE THREAD

**Issue by Ricky-Tigg** [openssl/openssl#19053]
_release:_ openssl-3.0.5

- _references:_ (outputs truncated)
```
$ openssl help speed
Usage: speed [options] [algorithm...]
(...)
 algorithm           Algorithm(s) to test (...)
$ man openssl-speed
 evp algo
  Use the specified cipher or message digest algorithm via the EVP interface. (...)
  To see the algorithms supported with this option, use (...)
  "openssl list -cipher-algorithms" command.
```
_Notes_: meaning of _EVP_ not documented thus unknown to beginning users. | Separator to be used for multi selection of algorithms undocumented and thus unknown.

Some of the supported algorithms:
```
$ openssl list -cipher-algorithms | grep -i chacha | grep -v @ | column
  ChaCha20		  ChaCha20-Poly1305
```
- Application

Hello. Support for selection of multiple algorithms in command-line appears not to be operational; illustration:
```
$ openssl speed -evp ChaCha20 ChaCha20-Poly1305
speed: Unknown algorithm ChaCha20-Poly1305
```

**Comment by t8m** [comment]
That's because the -evp applies only to the argument after it and you cannot specify it multiple times. The multiple algorithms as in the command usage applies only to the hardcoded algorithm names in the speed command.

**Comment by Ricky-Tigg** [comment]
I believe not to be wrong by stating that this information is not part of any sources of documentation either maintained by the project or in Wiki. If so, that would make it complementary, and yet such one that cannot be missing from man-pages, Obviously that lack is correlated by the lack of command option producing a list of those **hard-coded algorithm names**; At best currently, `openssl speed` has to be executed, and yet without involving the lowest value of custom-sized buffer, e.g `-bytes 16`, since some algorithms would failed to pass the test, yet filtered, to at last get such list. Finally, after resigning myself to using that last command, it revealed that none of the _Chacha_ algorithms were hard-coded.

**Comment by t8m** [comment]
Yes, the list of the hardcoded algorithm names should be printed as part of the speed command usage. That's a good point.

**Comment by marek22k** [comment]
Why has the feature request been closed now?

**Comment by t8m** [comment]
> _No action within a reasonable time frame_
> 
> From which to draw a conclusion.

Yes, the conclusion is the OpenSSL project does not have enough developers to implement all the various minor feature requests that keep coming into the Github issues for the project. The project welcomes and depends on external contributions of code for many such feature additions.
